### PR TITLE
KQP computation pattern cache serialized program

### PR DESCRIPTION
**Pull request by kitaisreal**
KQP computation pattern cache serialized program

### CODE REVIEW

**Review by va-kuznecov — APPROVED** [review]

File: ydb/library/yql/minikql/computation/mkql_serialized_program.h [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+/// Serialized program with precomputed hash
+class TSerializedProgram
+{
+public:
+    TSerializedProgram(TString data)
+        : Data(std::move(data))
+        , Hash(THash<TString>()(data))
+    {}
+
+    const TString & getData() const
```
Comment (va-kuznecov) [inline ydb/library/yql/minikql/computation/mkql_serialized_program.h:17]:
```suggestion
    const TString & GetData() const
```

File: ydb/library/yql/minikql/computation/mkql_serialized_program.h [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        return Data;
+    }
+
+    uint64_t getHash() const
```
Comment (va-kuznecov) [inline ydb/library/yql/minikql/computation/mkql_serialized_program.h:22]:
```suggestion
    uint64_t GetHash() const
```

File: ydb/library/yql/minikql/computation/mkql_serialized_program.h (hunk 1 above)
Comment (kitaisreal) [reply ydb/library/yql/minikql/computation/mkql_serialized_program.h]: Fixed.

File: ydb/library/yql/minikql/computation/mkql_serialized_program.h (hunk 2 above)
Comment (kitaisreal) [reply ydb/library/yql/minikql/computation/mkql_serialized_program.h]: Fixed.

**Pull request merged**